static/security.txt

```diff
@@ -1,5 +1,5 @@
 Contact: bugbounty@parity.io
-Preferred-Languages: en
+Preferred-Languages: english
```

Comment:
does this actually matter?

Reply:
Not much, just in case someone is unfamiliar with the abbreviation

Reply:
https://en.m.wikipedia.org/wiki/List_of_ISO_639-1_codes

This is very familiar to almost everyone as an industry standard, i would hope that it is rather obvious, and if this file was used in some engine to parse the data, it would include the iso code over the full length version.

Reply:
Thank You for your insight